_posts/2022-01-27-runway-to-sustainability-czi-funding.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Runway for sustainability: OLS awarded funding from the Chan Zuckerberg Initiative to power its next phase"
+authors:
+- bebatut
+- emmyft
+- malvikasharan
+- yochannah
+image: ../images/2022-01-27-ols-czi-funding.jpg
+photos:
+  name: Photo by Julian Hochgesang on Unsplash
+  url: https://unsplash.com/@julianhochgesang
+---
+
+OLS is absolutely delighted to share the news that as of January 2022, the Chan Zuckerberg Initiative has awarded OLS a total of $574,945.30 USD over the next two years. This grant will provide OLS with a truly valuable opportunity to improve sustainability for the organisation and community, and allow enough time to focus on planning for the longer term.
+
+Activities over 2022 and 2023 will include:
+
+1. **Four open research mentoring and training cohorts**: OLS-5 applications have closed, and we will run OLS-6 in the second half of 2022, followed by OLS-7 and OLS-8 in 2023.
+
+2. **Adding an additional optional Open Source Software module to the OLS curriculum**: - OLS will continue to cater to researchers and the open-research curious from all backgrounds, so this module will specifically be a skill-up enhancement for those who have software and code-related projects and wish to attend it.
+
+3. **Sustainability and business model planning**: Until recently, OLS was fully volunteer driven. For cohorts 3 and 4, we were able to offer small payments in the form of honoraria to some of our mentors and call facilitators. This will be a core offering over 2022 and 2023, as well as dedicated time to pay the OLS core team to work on OLS.
```

Comment:
```suggestion
3. **Sustainability and business model planning**: Until recently, OLS was fully volunteer-driven. For cohorts 3 and 4, we were able to offer small payments in the form of honoraria to some of our mentors and call facilitators. This will be a core offering over 2022 and 2023, as well as dedicated time to pay the OLS core team to work on OLS.
```